swarm/network/stream/syncer.go

```diff
@@ -28,8 +28,7 @@ import (
 )
 
 const (
-	// BatchSize = 2
-	BatchSize = 128
+	BatchSize = 8
```

Comment:
This is quite a dramatic change...are we aware of what it implies? Have we tried with other values? 16..32..64? In other words, is there a specific reason for it to be 8?